src/Sulu/Bundle/SecurityBundle/Tests/Functional/Controller/ResettingControllerTest.php

```diff
@@ -197,7 +199,8 @@ public function testResendEmailAction()
         $this->assertEquals($this->user3->getEmail(), $response->email);
 
         // asserting user properties
-        $user = $this->em->find('SuluSecurityBundle:User', $this->user3->getId());
+        $user = $client->getContainer()->get('doctrine')->getManager()
+            ->find('SuluSecurityBundle:User', $this->user3->getId());
```

Comment:
Same question for all changes in this method, shouldn't that be `$this->getContainer()`?